front/app/containers/Admin/projects/project/index.tsx

```diff
@@ -374,9 +378,7 @@ export class AdminProjectsProjectIndex extends PureComponent<
             project={project}
             phases={phases}
           />
-          <FeatureFlag name="project_management">
```

Comment:
Not sure why this was not integrated into the pre-existing tab system above - changed it now.